homeassistant/components/nam/const.py

```diff
@@ -46,7 +46,7 @@
 ATTR_SPS30_P4: Final = f"{ATTR_SPS30}{SUFFIX_P4}"
 ATTR_UPTIME: Final = "uptime"
 
-DEFAULT_UPDATE_INTERVAL: Final = timedelta(minutes=6)
+DEFAULT_UPDATE_INTERVAL: Final = timedelta(minutes=4)
```

Comment:
Thanks to the improvement of the retry logic, it is possible to shorten the update interval to 4 minutes.